app/javascript/components/ingredients/base-input-form.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <input
+    class="appearance-none block w-full bg-white text-gray-700 border border-gray-200 rounded py-3 px-4 mb-3 leading-tight focus:outline-none"
+    :id="id"
+    :type="type"
+    :placeholder="placeholder"
+  >
+</template>
+
+<script>
+export default {
+  props: {
+    id: { type: String, required: true },
+    type: { type: String, required: true },
+    placeholder: { type: String, required: true },
+  },
+};
+</script>
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline required at end of file but not found  eol-last